calico-enterprise/getting-started/bare-metal/about.mdx

```diff
@@ -65,50 +65,26 @@ To learn how to restrict traffic to/from hosts and VMs using Calico network poli
 
     Wait until the Tigera Manager and non-cluster Typha deployments reach the Available status before proceeding to the next step.
 
-2. Obtain the token for `tigera-noncluster-host` service account.
+1. Create a `kubeconfig` file for your non-cluster host or VM:
 
     ```bash
-    kubectl get secret -n calico-system tigera-noncluster-host -o jsonpath='{.data.token}' | base64 --decode
+    calicoctl nonclusterhost generate-config [--namespace=<NAMESPACE>] [--serviceaccount=<SERVICEACCOUNT>] [--certfile=<CERTFILE>] > kubeconfig
     ```
 
-3. Create a `kubeconfig` file for your non-cluster host or VM similar to the following:
```

Comment:
Step 2+3 is replaced by a new `calicoctl nonclusterhost generate-config` command.